boards/rfduino/Makefile.features

```diff
@@ -0,0 +1,2 @@
+FEATURES_PROVIDED += cpp
+FEATURES_PROVIDED += periph_uart periph_gpio periph_random periph_rtt periph_cpuid
```

Comment:
sh.. I think I removed and added the repository at my own git account. puh i am trying to fix it - maybe I can remove the 2312 pull and add a new one (with the same content and changes). I hope to google a solution today.